app/navigation/EngineeringNavigator.scala

```diff
@@ -28,10 +28,10 @@ class EngineeringNavigator @Inject()() extends Navigator {
 
   protected val routeMap: PartialFunction[Page, UserAnswers => Call] = {
     case TypeOfEngineeringPage => typeOfEngineeringOptions(NormalMode)
+    case ConstructionalEngineeringApprenticePage => constructionalEngineeringApprentice(NormalMode)
     case ConstructionalEngineeringList1Page => constructionalEngineeringList1(NormalMode)
     case ConstructionalEngineeringList2Page => constructionalEngineeringList2(NormalMode)
-    case ConstructionalEngineeringList3Page => constructionalEngineeringList3(NormalMode)
-    case ConstructionalEngineeringApprenticePage => constructionalEngineeringApprentice(NormalMode)
+    case ConstructionalEngineeringList3Page => _ => controllers.routes.EmployerContributionController.onPageLoad(NormalMode)
```

Comment:
pedantic but can we use import controllers.routes._ so that we can get rid of the long line.